pipelines/treatment/monitoramento/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - monitoramento
 
+## [1.2.4] - 2025-07-23
+
+### Alterado
+
+- Alterado o flow `MONITORAMENTO_VEICULO_MATERIALIZACAO` para adicionar a materialização do snapshot (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)